components/library/libArcus/Makefile

```diff
@@ -40,5 +41,4 @@ install: $(INSTALL_32_and_64)
 
 test: $(NO_TESTS)
 
-REQUIRED_PACKAGES += runtime/python-34
```

Comment:
runtime/python-35?

Reply:
Actually this is weird... Python is desactivated in this packages, see the CMAKE_OPTIONS.

Reply:
I replaced with $(PYTHON3_RUNTIME_PKG)